Installation/include/CGAL/config.h

```diff
@@ -728,4 +728,56 @@ typedef const void * Nullptr_t;   // Anticipate C++0x's std::nullptr_t
 /// @}
 #include <CGAL/license/lgpl.h>
 
+//----------------------------------------------------------------------//
+//  Function to define data directory
+//----------------------------------------------------------------------//
+#include <cstdlib>
+#include <string>
+#include <fstream>
+#include <iostream>
+
+namespace CGAL {
+
+// Returns filename prefixed by the directory of CGAL containing data.
+// This directory is either defined in the environement variable CGAL_DATA_DIR,
+// otherwise it is taken from the constant CGAL_DATA_DIR (defined in CMake),
+// otherwise it is empty (and thus returns filename unmodified).
+inline std::string get_data_file_path(const char* filename)
+{
+ std::string res;
+
+ const char *cgal_dir=std::getenv("CGAL_DATA_DIR");
+
+ if (cgal_dir!=nullptr)
+ { res=std::string(cgal_dir); }
+#ifdef CGAL_DATA_DIR
+ else
+ { res=std::string(CGAL_DATA_DIR); }
+#endif
+
+ if (!res.empty() && res.back()!='/')
+ { res+=std::string("/"); }
+
+ res+=std::string(filename);
+
+ // Test if the file exists, write a warning otherwise
+ std::ifstream f(res);
+ if (!f)
+ {
+   std::cerr<<"[WARNING] file "<<res<<" does not exist or cannot be read "
+            <<"(CGAL_DATA_DIR='"
+            <<(cgal_dir!=nullptr?cgal_dir:
+#ifdef CGAL_DATA_DIR
+               CGAL_DATA_DIR
+#else
+               ""
+#endif
+               )<<"')."<<std::endl;
+ }
+
+ return res;
+}
```

Comment:
Here is a more complete code, not tested:

```suggestion
inline std::string get_data_file_path(const char* filename)
{
 std::string cgal_dir_string;

 const char *cgal_dir=std::getenv("CGAL_DATA_DIR");
#ifdef CGAL_DATA_DIR
 if (cgal_dir==nullptr)
 { cgal_dir=CGAL_DATA_DIR; }
#endif
 if (cgal_dir!=nullptr)
 { cgal_dir_string=std::string(cgal_dir); }

 std::string res = cgal_dir_string;
 if (!res.empty() && res.back()!='/')
 { res+=std::string("/"); }

 res+=std::string(filename);

 // Test if the file exists, write a warning otherwise
 std::ifstream f(res);
 if (!f)
 {
   std::cerr<<"[WARNING] file "<<res<<" does not exist or cannot be read "
            <<"(CGAL_DATA_DIR='" << cgal_dir_string <<"')."<<std::endl;
 }

 return res;
}
```